instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/patches/querying.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActiveRecord
+      module Patches
+        # Module to prepend to ActiveRecord::Querying for instrumentation
+        module Querying
+          def find_by_sql(sql, binds = [], preparable: nil, &block)
+            tracer.in_span("#{self}.find_by_sql") do
```

Comment:
Maybe not for this first PR but eventually shouldn't this add the sql statement to the span as the [mysql instrumentation does](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/patches/client.rb#L57) ? 
Or is the idea to use both instrumentations together? (although in this case this one does not add much)

Reply:
>Or is the idea to use both instrumentations together?

The current direction we're taking with this instrumentation is that it would be used in addition to database instrumentation such as the mysql2 instrumentation.